components/BUILD.gn

```diff
@@ -31,9 +31,11 @@ test("brave_components_unittests") {
     "//brave/components/brave_shields/core/browser:unit_tests",
     "//brave/components/brave_shields/core/common:unit_tests",
     "//brave/components/brave_wallet/common:unit_tests",
+    "//brave/components/component_updater:unit_tests",
     "//brave/components/email_aliases:unit_tests",
     "//brave/components/json:unit_tests",
     "//brave/components/static_redirect_helper:unit_tests",
+    "//brave/components/update_client:unit_tests",
```

Comment:
shouldn't these new tests be `!ios && !android`?

Reply:
I believe the code in this PR gets used on Android, for example on `brave://components`. I suspect it doesn't get used on iOS but wonder if it's worth disabling there.